### PR TITLE
Feature: migrate from Vue CLI to Vite

### PR DESCRIPTION
**Pull request by NicolasRichel**
https://platform-dev-migrate-to-vite.bimdata.io/

For your information I made the migration based on the instruction in this guide:
=> https://vueschool.io/articles/vuejs-tutorials/how-to-migrate-from-vue-cli-to-vite/

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,50 +1,50 @@
 # Application URL
-VUE_APP_BASE_URL=http://localhost:8080
+VITE_BASE_URL=http://localhost:8080
```
Comment (Amoki) [inline .env.example]: Is there a way to keep old names for retro-compatibility with on-premises?

Comment (Amoki) [reply .env.example]: I found this on the Internet: https://github.com/ElMassimo/vite-plugin-environment/blob/main/README.md#loading-prefixed-variables

Comment (NicolasRichel) [reply .env.example]: I'll have a look, thanks !

**Review by Amoki — APPROVED** [review]

**Pull request merged**